docs/docs/start.md

```diff
@@ -5,8 +5,8 @@ title: ZeppelinOS
 
 Welcome to the ZeppelinOS Documentation Site!
 
-ZeppelinOS is a platform to develop, manage and operate smart contract
-applications in Ethereum.
+
+ZeppelinOS is a platform to develop, deploy and operate upgradeable smart contracts on Ethereum and every other EVM and eWASM-powered blockchain.
```

Comment:
ZeppelinOS is a platform to develop, deploy and operate upgradeable smart contract applications in Ethereum.

[Don't start with a technical, acronym-loaded phrase]

Reply:
It's a marketing decision, it's not really technical.

Reply:
ok, this prior version was written by someone marketing-savvy, but if you've checked with them, go ahead!

Reply:
I'm certain about this.